### DISCUSSION TITLE
posts/add-comment-service-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [imjhk03/imjhk03.github.io#154] [Comments]
# posts/add-comment-service-giscus/

오랜만에 사용하는 jekyll 테마 업데이트하면서 튜토리얼 글을 보니깐 댓글을 쓸 수 있는 것을 보았다. 보니깐 Giscus comments를 지원한 것이었다. 원래는 댓글 기능에 대해서 생각하지 않았는데, 신기하고 jekyll 테마가 지원하는 것을 하나씩 추가해 보는 게 재밌을 것 같아서 적용해 보았다. 처음으로 사용한 기능이어서 어떻게 적용했는지 간략하게 작성했다.

https://imjhk03.github.io/posts/add-comment-service-giscus/

**Comment by hunhee98** [comment]
안녕하세요!
혹시 페이지 소스 Minify 어떻게 하셨는지 여쭤봐도 될까요?
제 깃헙 블로그도 Joohee님과 같은 테마를 적용했는데 페이지 소스가 너무 난잡하더라고요.
Ruby도 jekyll도 처음인데다 웹 쪽과는 거리가 멀어서 어렵네요😂

jekyll 테마 쓰는 사람 중 한 줄로 줄여두신 분은 Joohee님 뿐이어서 댓글까지 남겨 여쭤봅니다.
늦었지만 새해 복 많이 받으세요!

**Reply by imjhk03** [reply]
안녕하세요~
늦은 새해 인사 감사합니다 ㅎㅎ walkerhilla님도 새해 복 많이 받으세요!

말씀해 주신 Minify는 저도 웹 분야랑 거리가 멀어서 정확하게 어떤 부분을 말씀하고 계시는지 잘 모르겠네요 😅

walkerhilla님 깃헙 블로그 리파지토리 보니깐 jekyll-theme-chirpy를 fork 해서 쓰신 것 같은데, 저는 [chirpy-starter](https://github.com/cotes2020/chirpy-starter) 템플릿을 썼습니다. 이거 때문에 페이지 소스가 차이 난 게 아닌가 생각이 듭니다. 혹시 댓글 단 내용이 이거랑 비슷한가요?

**Reply by hunhee98** [reply]
답변 감사합니다.
제가 Jekyll에 대해 무지한 채로 드린 질문이었네요^^;;
로컬 환경에서는 원래 압축되지 않은 소스가 나오는게 정상인 것 같습니다!
원격 리파지토리에 빌드하니까 Joohee님 블로그와 동일하게 압축된 페이지 소스로 서비스되네요!

감사합니다 :D

**Reply by imjhk03** [reply]
아하 그렇군요. 다행이네요! :)
